Makefile.am

```diff
@@ -114,6 +114,7 @@ bin_PROGRAMS = \
     tools/tpm2_startauthsession \
     tools/tpm2_startup \
     tools/tpm2_stirrandom \
+    tools/tpm2_testparms \
```

Comment:
Wait don't we want this as tpm2_testparams?

Reply:
Depends on what we want for the people using the tools : do we value more the matching with TPM2.0 API names or the end-user (whom may not be a TPM expert) comfort by providing meaningful names ?

Reply:
The lack of a is really messing with me. Since it's `Esys_TestParms` and I want tool to api mapping where possible, leave as is.